src/openforms/formio/formatters/formio.py

```diff
@@ -167,3 +169,31 @@ def format(self, component: Component, value: str) -> str:
             src=value,
             alt=text,
         )
+
+
+class CustomerProfileFormatter(FormatterBase):
+    def format(
+        self, component: CustomerProfileComponent, value: list[DigitalAddress]
+    ) -> str:
+        # Gather all filled-in addresses and possibly add "as preferred" suffix
+        addresses: list[str] = [
+            _("%s (as new preferred)") % address["address"]
```

Comment:
please use the consistent interpolation form like in the rest of the code base:


```suggestion
            _("{address} (as new preferred)").format(address=address["address"])
```
and always used named arguments so that translators have more context and can re-order arguments if that's necessary for the language being translated.